test-e2e/subdomain-detection.test.ts

```diff
@@ -1,19 +1,19 @@
-import { test } from '@playwright/test'
-import { testSubdomainRouting, expect } from './fixtures/config-test-fixtures.js'
+// import { test } from '@playwright/test'
```

Comment:
```suggestion
```

use the `test` fixture from ./fixtures/config-test-fixtures.js instead so it will use the correct paths & config setting, but without setting autoReload to true